clang/test/Driver/windows-compat-sycl-args.cpp

```diff
@@ -0,0 +1,9 @@
+// RUN: %clang -### --sycl -c %s 2>&1 | FileCheck %s
+// RUN: %clang -### -fsycl-device-only -fsycl %s 2>&1 | FileCheck %s
```

Comment:
-fsycl-device-only is the same as --sycl (also -fsycl-device-only overrides -fsycl).  You can test against just -fsycl if you want to verify offload.

Reply:
I think first command is enough to validate #541, so I think we can safely remove this line.